src/effects/VST/VSTEffect.cpp

```diff
@@ -1182,7 +1177,7 @@ int VSTEffect::ShowClientInterface(
       // Set some defaults since some VSTs need them...these will be reset when
       // normal or realtime processing begins
       mBlockSize = 8192;
-      DoProcessInitialize(mProjectRate);
+      vstValidator->GetInstance().DoProcessInitialize(mProjectRate);
```

Comment:
This change might be squashed onto step 4, making 4 more correct

Reply:
Ok, done in commit 4.5